rpc/params/params.go

```diff
@@ -792,6 +793,17 @@ type DeployerConnectionValues struct {
 	APIAddresses []string `json:"api-addresses"`
 }
 
+// IsControllerArg holds a machine name.
+type IsControllerArg struct {
```

Comment:
Remove this and keep `params.Entities` as the argument. That allows you to assert that a machine tag was passed, and retains the prior bulk behaviour.